client/web/src/user/settings/accessTokens/UserSettingsCreateAccessTokenCallbackPage.tsx

```diff
@@ -65,6 +65,14 @@ const REQUESTERS: Record<string, TokenRequester> = {
             'Please make sure you have VS Code running on your machine if you do not see an open dialog in your browser.',
         callbackType: 'new-tab',
     },
+    CODY_VSCODIUM: {
+        name: 'Cody - VS Code Extension',
```

Comment:
```suggestion
        name: 'Cody - VSCodium Extension',
```
Do we still call it VS Code?